.github/workflows/workflow.yml

```diff
@@ -53,6 +56,10 @@ jobs:
             expect-fail: true
     steps:
       - uses: actions/checkout@v2
+      - name: Insecure curl
+        if: ${{ matrix.os == 'macOS-latest' && matrix.cabal == '3.4.0.0-rc4' }}
+        shell: bash
+        run: echo insecure >> $HOME/.curlrc
```

Comment:
Note that this is necessary due to this issue:

https://gitlab.haskell.org/haskell/ghcup-hs/-/issues/102

Reply:
You could do something like, "if today is later than 2021-02-01, fail the build noisily" as a reminder to check whether it's fixed?

Reply:
Done